pkg/opssight/prometheus.go

```diff
@@ -215,6 +215,9 @@ func (p *SpecConfig) PerceptorMetricsConfigMap() (*components.ConfigMap, error)
 	if p.opssight.Spec.Perceiver.EnableImagePerceiver {
 		targets = append(targets, fmt.Sprintf("%s:%d", util.GetResourceName(p.opssight.Name, util.OpsSightName, p.names["image-perceiver"]), 3002))
 	}
+	if p.opssight.Spec.Perceiver.EnableArtifactoryPerceiver {
```

Comment:
Enable Quay Perceiver metrics is missing

Reply:
thanks, adding it now